addons/mail/static/src/model/model_field_relation_set.js

```diff
@@ -69,6 +69,9 @@ export class RelationSet {
                             const valB = followRelations(b, relatedPath);
                             switch (compareMethod) {
                                 case 'truthy-first': {
+                                    if (!valA && !valB) {
```

Comment:
a more natural way to write those conditions would be `valA === valB`

Reply:
Hello @seb-odoo 👋🏼 
I am not sure I get the rationale.
Those two assessments are not to same to me, but I guess I must be missing something.

Reply:
It's a compare method, if both values are the same nothing should be done.
The current diff relies on the way the rest of the case is written (checking only negative values).

Reply:
@seb-odoo , Done!!